apps/src/craft/craft.js

```diff
@@ -161,6 +176,42 @@ Craft.init = function (config) {
   Craft.level = config.level;
   Craft.skin = config.skin;
 
+  var levelTracks = [];
+  if (Craft.level.songs && musicMetadata) {
+    levelTracks = musicMetadata.filter(function(trackMetadata) {
+      return Craft.level.songs.indexOf(trackMetadata.name) !== -1;
+    });
```

Comment:
Now that I see this in two places, I wonder if this `getMetadataForTrackNames` operation should be pulled into some kind of static helper on MusicController.